package.json

```diff
@@ -129,41 +129,41 @@
     "@types/csurf": "^1.11.2",
     "@types/express-request-id": "^1.4.3",
     "@types/express-session": "^1.17.4",
-    "@types/http-errors": "^1.8.1",
-    "@types/jest": "^27.4.0",
-    "@types/jsonwebtoken": "^8.5.6",
-    "@types/node": "^16.11.12",
+    "@types/http-errors": "^1.8.2",
+    "@types/jest": "^27.4.1",
+    "@types/jsonwebtoken": "^8.5.8",
+    "@types/node": "^17.0.21",
```

Comment:
Should we upgrade the types when we're still using node 16?

(I imagine this may be mostly fine but I can also see some potential weirdness if they changed anything)

Reply:
durrrrr quite right